modules/container.sh

```diff
@@ -21,6 +21,14 @@ function run_with_chroot_setup() {
     shift
     # The rest of the arguments are the command arguments.
 
+    # Keep a copy of the resolv.conf, and copy in our own, we will need it during chroot
```

Comment:
O: This would have read a lot easier if it was a function wit ha name like - ensure DNS something something

Reply:
I did this, but then reverted it again (same for the `qemu-$arch-static` copy). It relies on variables I deduce in this function, so it becomes more messy than it's worth IMHO, since we have to then recalculate them four times (twice in setup functions, twice in teardown functions).